README.md

```diff
@@ -1,7 +1,12 @@
 # Open API Spex
 
+[![Elixir CI](https://github.com/open-api-spex/open_api_spex/actions/workflows/elixir.yml/badge.svg)](https://github.com/open-api-spex/open_api_spex/actions/workflows/elixir.yml)
 [![Build Status](https://travis-ci.com/open-api-spex/open_api_spex.svg?branch=master)](https://travis-ci.com/open-api-spex/open_api_spex)
```

Comment:
Optional: We can drop the travis-ci badge now that we've migrated to github actions.